notebooks/Getting_started_with_Forecast.ipynb

```diff
@@ -332,7 +316,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "forecast.describe_dataset_group(DatasetGroupName=datasetGroupName)"
+    "forecast.describe_dataset_group(DatasetGroupArn=datasetGroupArn)"
```

Comment:
Line 310 doesn't go with this. We should either remove that line, or mention that you can use update_dataset_group to update dataset group

Reply:
Update the line to better reflect the intent.